mo_net/samples/cbow.py

```
@@ -0,0 +1,491 @@
+                tuple(
+                    chain(
+                        sentence[i - context_size : i],
+                        sentence[i + 1 : i + context_size + 1],
+                    )
+                ),
+                sentence[i],
+            )
+            for sentence in tokenized_sentences
+            for i in range(context_size, len(sentence) - context_size)
+        ]
+    )
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)